docs/sandbox/MapboxVectorTilesApi.md

```diff
@@ -40,40 +43,67 @@ The feature must be configured in `router-config.json` as follows
       "minZoom": 12,
       "cacheMaxSeconds": 600
     },
+    // all rental places: stations and free-floating vehicles
     {
       "name": "citybikes",
-      "type": "VehicleRental", // all rental places: stations and free-floating vehicles
+      "type": "VehicleRental",
       "mapper": "Digitransit",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 60,
       "expansionFactor": 0.25
     },
+    // just free-floating vehicles
     {
       "name": "rentalVehicles",
-      "type": "VehicleRentalVehicle", // just free-floating vehicles
-      "mapper": "Digitransit",
+      "type": "VehicleRentalVehicle",
+      "mapper": "DigitransitRealtime",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 60
     },
+    // just rental stations
     {
       "name": "rentalStations",
-      "type": "VehicleRentalStation", // just rental stations
+      "type": "VehicleRentalStation",
       "mapper": "Digitransit",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 600
     },
+    // Contains just stations and realtime information for them
+    {
+      "name": "realtimeRentalStations",
+      "type": "VehicleRentalStation",
+      "mapper": "DigitransitRealtime",
+      "maxZoom": 20,
+      "minZoom": 14,
+      "cacheMaxSeconds": 60
+    },
+    // note this has the same name as the example with the Digitransit mapper,
+    // at some point we might want to do some clean up and rename this to
+    // realtimeVehicleParking but for now this remains like this for backwards
+    // compatibility.
```

Comment:
This is incorrect. The name of the layer can be freely chosen. You can have two layers with the same type but a different name.

Reply:
Yes the names can be freely chosen but I meant mostly for this documentation (and I guess people might use these names in practice). What should I write here?

Reply:
I tried rewording the comment here so it wouldn't leave too much room for misinterpretation https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/8f6ac4a92fa8c5f22e038c7af7767d6ad442e643

Reply:
I would either use another name in the example or make it clear that you have to choose one and cannot copy paste the entire file from here.

Reply:
I've updated it https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/e1c1d6b631ef920cdf4c97a60b22afdd3902829d